<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewHotkeyDialog</class>
 <widget class="QDialog" name="NewHotkeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotkey configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons.qrc">
    <normaloff>:/Icons/car.png</normaloff>:/Icons/car.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Shortcut:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditShortcut">
       <property name="placeholderText">
        <string comment="Insert hotkey"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxAction">
     <item>
      <property name="text">
       <string>single shot equation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>start script</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>stop script</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>change sheet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>run control continue</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>run control stop</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>run control next one</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>run control next xxx</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Formula:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFormula">
       <property name="placeholderText">
        <string comment="Insert data"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>lineEditShortcut</sender>
   <signal>returnPressed()</signal>
   <receiver>NewHotkeyDialog</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewHotkeyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewHotkeyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
